datalad/distribution/install.py

```diff
@@ -234,7 +236,7 @@ def __call__(
                     # should be necessary here, all done by code further
                     # down that deals with an install from an actuall `source`
                     # any necessary fixes should go there too!
-                    # TODO generator: possibly adjust refds
+                    r.update(dict(refds=refds_path))
```

Comment:
`r['refds'] = refds_path` would be simpler and faster:

```
% python -m timeit -s "d = {}" "d['k'] = 'v'"
10000000 loops, best of 5: 27.1 nsec per loop
% python -m timeit -s "d = {}" "d.update(dict(k='v'))"
1000000 loops, best of 5: 252 nsec per loop
```

Reply:
Done